server/api/crud/runs.py

```diff
@@ -468,3 +453,48 @@ async def _post_delete_run(project, uid):
                 project,
                 uid,
             )
+
+    @staticmethod
+    def _get_artifacts_from_uris(db_session, project, run):
+        artifacts = []
+        artifact_uris = run.get("status", {}).get("artifact_uris", {})
+        for key, uri in artifact_uris.items():
```

Comment:
Not needed since its parsed from the URI
```suggestion
        for _, uri in artifact_uris.items():
```